tool/missing-baseruby.bat

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+echo "executable host ruby is required.  use --with-baseruby option."
```

Comment:
One more for `cmd.exe`, the command will be passed `echo` as written including quotes, and `echo` prints the quotes too.

```suggestion
echo executable host ruby is required.  use --with-baseruby option.
```